### ISSUE TITLE
Print message after retrieving test LPT

### ISSUE THREAD

**Issue by blake41** [livepeer/go-livepeer#500]
i'm on livepeer rinkeby CL, when i select get test LPT, nothing appears to happen, then i selected get node status and realized i had test LPT, might be nice to have some kind of message print to the screen letting the user know it worked

**Comment by gt12889** [comment]
I opened [PR #4039](https://github.com/livepeer/go-livepeer/pull/4039) reporting successful test-token requests only after a 2xx response, with httptest coverage. It is awaiting maintainer review.

**Comment by dk5488** [comment]
I’d love to help add a confirmation message for the “get test LPT” action—could you assign this to me? Thanks!
